samza-core/src/main/scala/org/apache/samza/util/CoordinatorStreamUtil.scala

```diff
@@ -37,13 +39,12 @@ object CoordinatorStreamUtil extends Logging {
     * Given a job's full config object, build a subset config which includes
     * only the job name, job id, and system config for the coordinator stream.
     */
-  def buildCoordinatorStreamConfig(config: Config) = {
+  def buildCoordinatorStreamConfig(config: Config): MapConfig = {
     val jobConfig = new JobConfig(config)
-    val buildConfigFactory = jobConfig.getCoordinatorStreamFactory();
+    val buildConfigFactory = jobConfig.getCoordinatorStreamFactory
```

Comment:
nit: I'd recommend leaving the parentheses as is for convention. all java method call inside scala has explicit parentheses even in the absence of parameters.

Reply:
reverted the change to keep consistency.